gateway-cm-integration/src/main/java/org/apache/knox/gateway/topology/discovery/advanced/AdvanceServiceDiscoveryConfigurationMessages.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
There is AdvancedServiceDiscoveryConfig, then this is named AdvanceServiceDiscoveryConfigurationMessages.
Not a big deal at all, but they differ ("Advanced" vs "Advance").